core/idl/wallet/common/wallet.djinni

```diff
@@ -43,6 +43,7 @@ DerivationPath = interface +c {
 OperationType = enum {
     send;
     receive;
+    none;
```

Comment:
What does it mean for an operation to be of no type?

Reply:
Why we don't want to make this kind of operation as receive?

Reply:
As explained in comments, we can have transactions made by someone else which leads to side effects e.g. changing our balance of erc20 token.
It's not a receive because it's not a value transfer toward our address, plus later we might have other side effects that affect account's address without any notion of transfer (e.g. internal transactions in general)

Reply:
Hm, I don’t get that _made by someone else_ part. If it’s a transfer, can we not make that an `OperationType::Transfer` type?

Reply:
Basically I can interact with a smart contract (so transaction from `me` to `smartContract`) and this smart contract is sending events or making side effects on your account without being a transfer any kind
At the end from our clients side most important is to know if it's a receive, send or `else` they don't care about a specific type (it's mostly important for tx history, balance history)